src/foam/core/AbstractArrayPropertyInfo.java

```diff
@@ -67,4 +73,36 @@ public void toXML (FObject obj, Document doc, Element objElement) {
       prop.appendChild(nestedProp);
     }
   }
+
+  @Override
+  public void setStatementValue(IndexedPreparedStatement stmt, FObject o) throws java.sql.SQLException {
+    Object obj = this.get(o);
+    if ( obj == null ) {
+      stmt.setObject(null);
+      return;
+    }
+    Object[] os = (Object[]) obj;
+    java.lang.StringBuilder sb = new java.lang.StringBuilder();
+    int length = os.length;
+    if ( length == 0 ) {
+      stmt.setObject(null);
+      return;
+    }
+    for ( int i=0; i < length; i++) {
```

Comment:
FOAM Coding standard requires this line be:
for ( int i = 0 ; i < length ; i++ ) {